roles/container-engine/cri-o/templates/crio.conf.j2

```diff
@@ -64,7 +64,11 @@ file_locking = true
 # This is a mandatory setting as this runtime will be the default one
 # and will also be used for untrusted container workloads if
 # runtime_untrusted_workload is not set.
+{% if ansible_os_family == "ClearLinux" %}
 runtime = "/usr/bin/runc"
+{% else %}
+runtime = "/usr/sbin/runc"
```

Comment:
You are changeing the default `/usr/bin/runc` to `/usr/sbin/runc` . Is this correct, since the template until now have been using `/usr/bin/runc`, but this will now change so only ClearLinux will use that path, and everything else will use `/usr/sbin/runc`

Reply:
@woopstar that's right, the path depends on the distro.

```
$ cat /etc/os-release | grep ID=
ID="centos"
VERSION_ID="7"
$ which runc
/usr/sbin/runc
```

```
$ cat /etc/os-release | grep ID=
ID=clear-linux-os
VERSION_ID=28510
$ which runc
/usr/bin/runc
```